BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -121,6 +121,77 @@ extension Element {
         )
     }
 
+    /// Wraps the receiver in an accessibility element with the provided values.
```

Comment:
Could we provide a better name for this method? Eg, the only difference here is that the method passes through `nil`, which we could also do through extending the base method with optional parameters. Eg, could we instead give this a better name for the use cases where these parameters are always nil? Like `accessibilityButton` or something.